js/background.js

```diff
@@ -50,9 +50,11 @@ function Background() {
             epoch = isDST ? estEpoch - oneHour : estEpoch,
             timeSinceEpoch = new Date().getTime() - epoch,
             majorVersion = Math.ceil(timeSinceEpoch / oneWeek),
-            minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay);
+            minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay), 
+            atb = 'v' + majorVersion + '-' + minorVersion;
 
-        localStorage['atb'] = 'v' + majorVersion + '-' + minorVersion;
+        localStorage['atb'] = atb;
+        localStorage['set_atb'] = atb;
```

Comment:
I don't think we want to set this here. The current purpose of this field is to store when we've successfully pulled the atb param off the DOM. Once it's set we never look to the DOM again, preventing it from somehow being overwritten and changed.